webapp/channels/src/components/user_list.tsx

```diff
@@ -16,7 +15,7 @@ import UserListRow from './user_list_row';
 type Props = {
     rowComponentType?: React.ComponentType<any>;
     length?: number;
-    actions?: ReactNode[];
+    actions?: Array<React.ComponentType<any>>;
```

Comment:
Does it make sense to specify what the type of these components should be? Looks like we're always passing in the same props so we should know what the type is.

Also might be out-of-scope.